_posts/2023-05-12-data-center-maintenance.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Data Center Annual Maintenance Scheduled June 5-12, 2023
+state: active
+severity: 0
+---
+
+Due to a scheduled power outage at the MGHPCC data center, MOC services will be unavailable from Tuesday, June 5th, 2023 at 6 PM through Friday, June 12th, 2023 at 12 AM.
```

Comment:
```suggestion
Due to the annually scheduled maintenance power outage at the MGHPCC data center, MOC services will be unavailable from Tuesday, June 5th, 2023 at 6 PM through Friday, June 12th, 2023 at 12 AM.
```

Many people are aware the facility shuts down every year so it would be good to indicate that this is indeed that event.